### PR TITLE
docs: explain how to find a peer's multiaddr

### LINKED ISSUE #9: How to use IPFS check with a local IPFS node
It would be useful if [IPFS Check](https://ipfs-check.on.fleek.co) had some instructions and a link to a docs page about how to find your multiadder via the webUI as an individual running a local IPFS node (ex via IPFS Desktop).

## How to:
1. Open IPFS WebUI "Status" page via IPFS Desktop drop down or by visiting "http://127.0.0.1:5001/ipfs/bafybeihcyruaeza7uyjd6ugicbcrqumejf6uf353e5etdkhotqffwtguva/#/" (with default config settings)
2. Click the "Advanced" drop down to see more detailed node settings
3. Choose the top entry in your list of "Addresses" (these are the various multiaddrs via which your node is addressable)
4. Insert this address into the "multiaddr" text box

### PR DESCRIPTION
**Pull request by aschmahmann**
fixes #9

### CODE REVIEW

File: web/index.html [hunk 1]
```diff
@@ -38,6 +38,23 @@ <h1 class="dib pa3 ma0 lh-tight">
         </form>
     </section>
     <section class="mw8 center lh-copy dark-gray pv4 ph2 ph4-ns">
+        <h2 class="f4">Where do I find my multiaddr?</h2>
+        <ul>
+            <li class="pb2">
+                <strong>Using IPFS Desktop</strong>
```
Comment (aschmahmann) [inline web/index.html:44]:
```suggestion
                <strong>Using IPFS Desktop or IPFS WebUI</strong>

```

File: web/index.html [hunk 2]
```diff
@@ -38,6 +38,24 @@ <h1 class="dib pa3 ma0 lh-tight">
         </form>
     </section>
     <section class="mw8 center lh-copy dark-gray pv4 ph2 ph4-ns">
+        <h2 class="f4">Where do I find my multiaddr?</h2>
+        <ul>
+            <li class="pb2">
+                <strong>Using IPFS Desktop or IPFS WebUI</strong>
+
+                <ul>
```
Comment (aschmahmann) [inline web/index.html:46]:
```suggestion
                <strong>Using IPFS Desktop or IPFS WebUI</strong>
                <ul>

```

File: web/index.html [hunk 3]
```diff
@@ -38,6 +38,24 @@ <h1 class="dib pa3 ma0 lh-tight">
         </form>
     </section>
     <section class="mw8 center lh-copy dark-gray pv4 ph2 ph4-ns">
+        <h2 class="f4">Where do I find my multiaddr?</h2>
+        <ul>
+            <li class="pb2">
+                <strong>Using IPFS Desktop or IPFS WebUI</strong>
+                <ul>
+
+                    <li>Open IPFS WebUI "Status" page via IPFS Desktop drop down or by visiting "http://127.0.0.1:5001/webui" (with default config settings)
```
Comment (aschmahmann) [inline web/index.html:47]:
```suggestion
                <strong>Using IPFS Desktop or IPFS WebUI</strong>
                <ul>
                    <li>Open IPFS WebUI "Status" page via the IPFS Desktop menu or by visiting "http://127.0.0.1:5001/webui" (when using the default config settings)
```

File: web/index.html [hunk 4]
```diff
@@ -38,6 +38,23 @@ <h1 class="dib pa3 ma0 lh-tight">
         </form>
     </section>
     <section class="mw8 center lh-copy dark-gray pv4 ph2 ph4-ns">
+        <h2 class="f4">Where do I find my multiaddr?</h2>
+        <ul>
+            <li class="pb2">
+                <strong>Using IPFS Desktop or IPFS WebUI</strong>
+                <ul>
+                    <li>Open the IPFS WebUI "Status" page via the IPFS Desktop menu or by visiting "http://127.0.0.1:5001/webui" (when using the default config settings)
```
Comment (BigLep) [inline web/index.html:46]: Renders fine for me but is missing trailing </li>

Comment (aschmahmann) [reply web/index.html:46]: Thanks for the catch 🙏

**Review by momack2 — COMMENTED** [review]
awesome!! thank you!

**Pull request merged**